build/Dockerfile

```diff
@@ -98,20 +92,31 @@ RUN dnf config-manager \
 
 ENV LIBRARY_PATH="$CUDA_HOME/lib64/stubs"
 
-FROM cuda-devel as python-installations
+## Python dep management / common files for dev & release #####
+FROM cuda-devel as files-common
 
 ARG WHEEL_VERSION
 ARG USER
 ARG USER_UID
-
-## Enable Aimstack if requested via ENABLE_AIM set to "true"
-ARG ENABLE_AIM=false
+ARG ENABLE_AIM
```

Comment:
Probably no longer need this build arg in this stage since its used in later stages unless we want to setup the aim_profile in this stage and copy it in other stages. But I think you'd still need the if block so doesn't save too much